Source/ConvolutionFunctions/arm_convolve_1x1_s8_fast.c

```diff
@@ -21,8 +21,8 @@
  * Title:        arm_convolve_1x1_s8_fast.c
  * Description:  Fast s8 version of 1x1 convolution (non-square shape)
  *
- * $Date:        17 October 2023
- * $Revision:    V.3.3.0
+ * $Date:        25 October 2023
+ * $Revision:    V.3.4.0
```

Comment:
Going by https://semver.org/ and since this is a bug fix, it is the 'patch' version that has to be incremented here. i.e 3.3.1